app/src/main/scala/io/codegalaxy/app/question/QuestionScreen.scala

```diff
@@ -160,38 +168,67 @@ object QuestionScreen extends FunctionComponent[QuestionScreenProps] {
 
     val container: Style = new ViewStyle {
       override val flex = 1
-      override val margin = 5
-      override val padding = 5
+//      override val margin = 5
+//      override val padding = 5
       override val marginBottom = 0
       override val paddingBottom = 0
+      override val backgroundColor = "#e3e3e2"
+    }
+    val innerContainer: Style = new ViewStyle {
+      override val flex = 1
+    }
+    val textWrapper = new ViewStyle {
+      override val borderWidth = 1
+      override val borderRadius = 20
+      override val borderColor = Color.white
+      override val margin = 10
+      override val padding = 20
+      override val backgroundColor = Color.white
     }
     val choiceGroup: Style = new ViewStyle {
-      override val alignSelf = AlignSelf.center
-      override val margin = 5
+      override val alignSelf = AlignSelf.`flex-start`
+      override val margin = 10
       override val paddingVertical = 5
       override val paddingLeft = 5
       override val paddingRight = 25
     }
+    val choiceOption: Style = new ViewStyle {
+      override val flexDirection = FlexDirection.row
+      override val padding = 10
+
+      override val borderWidth = 1
+
+      override val backgroundColor = Color.white
```

Comment:
I guess we should make this style theme-dependent and choose one depending on current theme.